### ISSUE TITLE
The networkFee() in AssetTransfer can support double input or string to avoid error

### ISSUE THREAD

**Issue by chenquanyu** [neow3j/neow3j#80]
```
AssetTransfer at = new AssetTransfer.Builder(neow3j)
                //.networkFee(new BigDecimal(0.1))  will cost more than 0.1 gas
                .networkFee(new BigDecimal("0.1"))
                .build();
```
I think neow3j can support double input to avoid this.

**Comment by claudemiller** [comment]
@chenquanyu, you mean we shouldn't allow the `networkFee()` method to take a BigDecimal as argument, because the user might make the mistake that you show in your code example?

**Comment by chenquanyu** [comment]
Yes, it's better to add networkFee like this:
```
.networkFee(0.1)
```

**Comment by gsmachado** [comment]
In my opinion, we could support both. But, then more internal checks are necessary to be made -- to ensure correctness.

**Comment by claudemiller** [comment]
Resolved in release 2.1.0
